src/test/ui/argument-suggestions/basic.stderr

```diff
@@ -0,0 +1,92 @@
+error[E0308]: arguments to this function are incorrect
+  --> $DIR/basic.rs:20:5
+   |
+LL |     invalid(1.0);
+   |     ^^^^^^^
+   |
+note: expected `u32`, found floating-point number
+  --> $DIR/basic.rs:20:13
+   |
+LL |     invalid(1.0);
+   |             ^^^
+note: function defined here
+  --> $DIR/basic.rs:13:4
+   |
+LL | fn invalid(_i: u32) {}
+   |    ^^^^^^^
+
+error[E0061]: this function takes 0 arguments but 1 argument was supplied
+  --> $DIR/basic.rs:21:5
+   |
+LL |     extra("");
+   |     ^^^^^ -- argument unexpected
+   |
+note: function defined here
+  --> $DIR/basic.rs:14:4
+   |
+LL | fn extra() {}
+   |    ^^^^^
+help: remove the extra argument
+   |
+LL |     extra();
+   |     ~~~~~~~
```

Comment:
Ideally, given the conversation we had yesterday, I would like this to be, but it doesn't have to be in this PR

```suggestion
help: remove the extra argument
   |
LL -     extra("");
LL +     extra();
   |
```

Reply:
Let's leave this for followup.